lib/makara/middleware.rb

```diff
@@ -89,6 +89,21 @@ def makara_values(env)
       [nil, nil]
     end
 
+    # determine how lone the cookie TTL should last
+    # this needs to be done huristically, as we cannot be sure which connection 
+    # would have had the longest timeout from this middleware
+    def cookie_ttl
```

Comment:
i don't see the new `cookie_ttl` setting here. did you mean for the second one to be that?
Should this use the `Makara::ConfigParser` ?